AsyncDisplayKit/ASMapNode.h

```diff
@@ -0,0 +1,45 @@
+/* Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import <AsyncDisplayKit/AsyncDisplayKit.h>
+#import <MapKit/MapKit.h>
+@interface ASMapNode : ASControlNode
+- (instancetype)initWithCoordinate:(CLLocationCoordinate2D)coordinate NS_DESIGNATED_INITIALIZER;
+/**
+ This is the snapshot shot image node, this will be hidden (but not nil) when .liveMap = YES
+ */
+@property (nonatomic, readonly) ASImageNode *mapImage;
```

Comment:
What use cases do you envision for exposing this?  If it is needed, we should probably call it mapImageNode (for a moment I assumed it was a UIImage).

I think it might be more useful to have a UIImage called mapImage with no access to the ASImageNode, but it might be better to provide no access at all.  Providing access requires that we also provide a means to tell when the image is done loading / complete, although I would argue that's one of the most important things we could provide (moreso than the image actually).  Technically it may be possible to determine this with subnodeDidFinishDisplay: or some similarly named method, but I think as-is this is tricky because the ASImageNode is an internal subnode of the ASControlNode.

One nifty idea: how about the ASMapNode subclasses ASImageNode directly, and we remove this entirely?  Then you'd have incidental access to the ASImageNode (although, still wondering what use cases exist for that), we'd have fewer nodes / views overall, and you could still create the live map to put inside of it / cover any image being displayed.  ASImageNode is an ASControlNode, too.